library/core/src/sync/atomic.rs

```diff
@@ -107,6 +107,33 @@
 //! }
 //! ```
 //!
+//! Although the most convenient way to use an atomic type is to wrap it as an `Arc`,
+//! it is possible to share an atomic type between threads without `Arc`.
+//!
+//! ```
+//! use std::sync::atomic::{AtomicUsize, Ordering};
+//! use std::thread;
+//!
+//! fn main() {
+//!     let data: [usize; 20] = b"Fearless Concurrency".map(Into::into);
+//!
+//!     let result = AtomicUsize::new(0);
+//!     // Create a scope to process sum in parallel.
+//!     thread::scope(|s| {
+//!         // choose the size (data.len()+N-1)/N to divide the data into N chunks
```

Comment:
You could just use `.chunks(7)` or `.chunks(8)` instead, or make this a small closure, something like
```rust
let divide_into = |n| (data.len() + n - 1) / n;
```